pdns/recursor_cache.cc

```diff
@@ -371,6 +372,7 @@ int MemRecursorCache::doWipeCache(const DNSName& name, bool sub, uint16_t qtype)
 	break;
       if(iter->d_qtype == qtype || qtype == 0xffff) {
 	count++;
+        d_bytes -= iter->d_bytes;
```

Comment:
The indentation seems wrong here

Reply:
mixed tabs and spaces that github gets confused about. pushed.